src/stream.rs

```diff
@@ -83,6 +83,26 @@ impl<T: rt::Write + rt::Read + Unpin> rt::Write for MaybeHttpsStream<T> {
         }
     }
 
+    #[inline]
+    fn poll_write_vectored(
```

Comment:
nit: Can we keep the fn order in the impl matching the order from the `rt::Write` trait? e.g. poll_write, poll_flush, poll_shutdown, and then the two new fns: is_write_vectored followed by `poll_write_vectored`.

Reply:
Done!

Reply:
Thanks!